src/main/java/org/semver4j/RangesListFactory.java

```diff
@@ -1,17 +1,23 @@
 package org.semver4j;
 
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
 /**
  * Class for create a {@link RangesList} object.
  */
 public class RangesListFactory {
-    public static RangesList create(String range) {
+    @NotNull
+    public static RangesList create(@NotNull final String range) {
         return new RangesString().get(range);
     }
 
     /**
      * @since 4.2.0
      */
-    public static RangesList create(RangesExpression rangeExpressions) {
+    @NotNull
+    @ApiStatus.AvailableSince("4.2.0")
```

Comment:
Please use static import.

Reply:
Be carefull with static import. If it improves code readabillity is ok, otherwise it is not..
`Math.cos()` to `cos()` will looks good
```java
return hypotenuse * cos(angle);
```
But...
```java
address = getByName(name);
```
does not looks good as
```java
address = InetAddress.getByName(name);
```

Reply:
In that case make sense to use static import, but yes I'm aware about this pitfall.
That also applies e.g. `Optional.of()` or just `of()` - that could be sneaky.